src/cdk/v2/destinations/bluecore/data/bluecoreSubscriptionEventConfig.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "destKey": "properties",
+    "sourceKeys": "properties",
+    "metadata": {
```

Comment:
Do we have doc link which suggest we can send custom properties to subscription event apart from the already mapped ones. Its not clear here https://help.bluecore.com/en/articles/6786828-events-api#h_66485dc4cd